### PR TITLE
Translate strings and privacy policy to Spanish.

### PR DESCRIPTION
**Pull request by waninkoko**
This is a translation of the strings and privacy policy to Spanish. The translation of the help section is still missing (to be done soon).

### CODE REVIEW

File: app/src/main/res/values-es/about.xml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="about_title">Acerca De</string>
+    <string name="about_close">Cerrar</string>
+    <string name="about_version">Soy monerujo %1$s (%2$d)</string>
+
+    <string name="donation_text">
+        \"¡Donar bastardos desagradecidos!\"
```
Comment (anhdres) [inline app/src/main/res/values-es/about.xml]: ¡Donad, bastardos ingratos! *(sounds more like old spanish from spain or pirates)*
¡Donen, bastardos desagradecidos! *(sounds more like generic latin spanish)*

File: app/src/main/res/values-es/about.xml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    </string>
+
+    <string name="donation_address_label">Dirección de Donaciones</string>
```
Comment (anhdres) [inline app/src/main/res/values-es/about.xml]: Dirección para donaciones *(literally address for donations)*

**Comment by EarlOfEgo** [comment]
Muy bien, muchas gracias. Will you add the missing translations of the help section as well?

File: app/src/main/res/values-es/about.xml (hunk 2 above)
Comment (waninkoko) [reply app/src/main/res/values-es/about.xml]: Definitely this is a better translation. Thanks!

File: app/src/main/res/values-es/about.xml (hunk 1 above)
Comment (waninkoko) [reply app/src/main/res/values-es/about.xml]: I agree with this. The last one would suit better.

**Comment by waninkoko** [comment]
@EarlOfEgo I hope I can have something ready for today.

**Comment by waninkoko** [comment]
Help section translated!

File: app/src/main/res/values-es/about.xml (hunk 1 above)
Comment (m2049r) [reply app/src/main/res/values-es/about.xml]: we want the pirates!

Comment (waninkoko) [reply app/src/main/res/values-es/about.xml]: So... do I bring back the pirates?

Comment (m2049r) [reply app/src/main/res/values-es/about.xml]: claro que si :)

Comment (waninkoko) [reply app/src/main/res/values-es/about.xml]: Remember the pirates? They're back, in pog form

**Review by EarlOfEgo — APPROVED** [review]
No idea about the Spanish, but LGTM

**Comment by m2049r** [comment]
Thank you!

**Pull request merged**